### DISCUSSION TITLE
Rearrange dimensions of a matrix

### DISCUSSION THREAD

**Discussion by medihack** [josdejong/mathjs#2717] [Ideas]
It would be nice to have a simple method to rearrange the dimensions of a (multidimensional) matrix.
Something like [permute in Matlab](https://de.mathworks.com/help/matlab/ref/permute.html) or [transpose in Numpy](https://docs.scipy.org/doc/numpy/reference/generated/numpy.transpose.html).
Any plans, or is this somehow already implemented?

**Comment by josdejong** [comment]
Thanks for your suggestion Kai. This isn't yet implemented but let's make it a feature request :)
